packages/api/src/models/customAnimalBreedModel.js

```diff
@@ -49,6 +50,20 @@ class CustomAnimalBreed extends baseModel {
       additionalProperties: false,
     };
   }
+
+  static async getBreedsByFarmAndTypeBreedPairs(farm_id, typeBreeds) {
```

Comment:
I think this should receive `trx`. Otherwise I think this could create a 3rd transaction in this one request.